ui/partials/perf/comparechooserctrl.html

```diff
@@ -35,7 +35,7 @@
         <div class="card-header">New</div>
         <div class="card-body">
           <label for="new-project-selector">Project</label>
-          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips();getPreviousRevision()"></select>
```

Comment:
This function doesn't exist any more (https://github.com/mozilla/treeherder/commit/d7d2c978c4c63365d2716d1ea20ca7c471393136#diff-94cd4f72cb47532b08533943ecb773e2L71).

Not sure why this didn't result in an error? @wlach ?